packages/simcore-sdk/src/simcore_sdk/node_ports/filemanager.py

```diff
@@ -27,16 +27,20 @@ class ClientSessionContextManager:
     # See https://github.com/ITISFoundation/osparc-simcore/issues/1098
     #
     def __init__(self, session=None):
-        self.active_session = session or ClientSession()
+        self.active_session = session or ClientSession(
+            timeout=ClientTimeout(total=None, connect=1, sock_connect=1)

```

Comment:
Yes, you are correct. The idea is: files may take long to upload and download and we'd like to make sure that the connections are successful or fail fast.

Reply:
Update: there is no timeout for the total duration of the connection!

Reply:
We do not care about 5 seconds. Before it was 5 minutes total. It can be bumped up to 5.

I am expecting a handshake to finish in the first second. This should have nothing to do with the payload it is transferring, but yes we can raise it.